force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +103,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> words = sentence.split (' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>();
 
         // Add each word to the set
+        for (String word : words){
+            uniqueWords.add(word);
+        }
+        //Convert the set back to a list because the following String.join method will require a list as the first argument, not a set
+        List<String> uniqueWordsList = new List<String>(uniqueWords);
```

Comment:
With regard to your comment (on line 115) about the `join()` method requiring a list, in actuality, it will accept a set. But while we could use a set in this case, we need to remember that a set is unordered and we don't know exactly how the words will be rejoined. It turns out (when using a set) the words are rejoined in their original placement in the sentence. (I think this is because the set was originally created by casting it from a list -- but I could be wrong.)